cli/slsa-verifier/main_test.go

```diff
@@ -491,34 +493,82 @@ func Test_runVerifyArtifactPath(t *testing.T) {
 				artifactPath := filepath.Clean(filepath.Join(TEST_DIR, v, tt.artifact))
 				provenancePath := fmt.Sprintf("%s.intoto.jsonl", artifactPath)
 
-				cmd := verify.VerifyArtifactCommand{
-					ProvenancePath:      provenancePath,
-					SourceURI:           tt.source,
-					SourceBranch:        tt.pbranch,
-					BuilderID:           tt.pbuilderID,
-					SourceTag:           tt.ptag,
-					SourceVersionTag:    tt.pversiontag,
-					BuildWorkflowInputs: tt.inputs,
+				// TODO(#258): invalid builder ref.
+				semver := path.Base(v)
+				// For each test, we run 4 sub-tests:
```

Comment:
For tests whos error expects a build mismatch, this won't (correctly) fail for cases 1-3. 

Maybe you should switch on whether pbuilderID was specified? In those cases we are explicitly testing pBuilderID

Reply:
IIUC, only the tests for v0.0.2 use the `@main` builder, the one >= v1 seem to all use a proper `@vx.y.z`, so the build mismatch tests seem to be working. I'll double check.